### ISSUE TITLE
'to yield exchange satisfying' not documented

### ISSUE THREAD

**Issue by msiebuhr** [unexpectedjs/unexpected-express#17]
https://github.com/unexpectedjs/unexpected-express/blob/33d9d53acab96169fa4385a431684a9bf92f6964/lib/unexpectedExpress.js#L47

Comment says *Please prefer this one because it does use 'to satisfy' semantics*, so I guess it is important :)

**Comment by papandreou** [comment]
Damn, there isn't even a documentation site for unexpected-express. It's probably the plugin that I use most often myself, maybe except for unexpected-sinon. Skomagerens børn går i hullede sko...

**Comment by msiebuhr** [comment]
http://unexpected.js.org/unexpected-express/ was what I had in mind...

**Comment by papandreou** [comment]
Right, that's there, but there's no structured docs for the individual assertions (yet).

You're right, we should change the examples to use `to yield exchange satisfying`. If you have the time, the source is here: https://github.com/unexpectedjs/unexpected-express/blob/master/documentation/index.md

**Comment by joelmukuthu** [comment]
From the code, I'm guessing `to yield exchange exhaustively satisfying` is not supported?

**Comment by papandreou** [comment]
@joelmukuthu Correct! Have you found yourself needing that?

**Comment by joelmukuthu** [comment]
I'm introducing unexpected-express at my job and I'd like to make it possible to have exhaustive satisfactions for the new-comers (possibly to avoid omissions). I actually don't know how it would work, having to describe everything in the req/res but I just wanted to know if it's supported. Don't need it per se

**Comment by papandreou** [comment]
The exhaustiveness would only apply to the `response` part, as the `request` part is specifying the input, which per definition is exhaustive. Right?

**Comment by joelmukuthu** [comment]
Indeed, correct!

**Comment by papandreou** [comment]
Well, it would be a useful addition, and probably not very hard. PR welcome :)

**Comment by joelmukuthu** [comment]
Cool, will find some time to try it out :)
